### DISCUSSION TITLE
Galaxy Brain blabla

### DISCUSSION THREAD

**Discussion by lueluew** [Schweinepriester/github-profile-achievements#335] [Q&A] [answered]
this' a question

**Comment by bacca87** [comment, marked as answer]
and this is the answer

**Comment by WHAHA-HA** [comment]
Actually, I can achieve Quickdraw and Pull Shark badges with ease, but no Galaxy Brain.
PS: It'd be appreciated if you could mark my answer

**Reply by miladkhalighi** [reply]
Do you need an answer?

**Comment by Valerio-boi** [comment]
And this is the other answer

**Comment by miladkhalighi** [comment]
And also this is my question

**Comment by YSD98** [comment]
I want a galaxy brain Badge
